uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/widgets/ConfirmPopup.java

```diff
@@ -36,27 +39,70 @@
     @DataField("confirm-title")
     Span modalTitle;
 
+    @Inject
+    @DataField("confirm-inline-notification")
+    InlineNotification confirmInlineNotification;
+
     @Inject
     @DataField("confirm-message")
     Span modalConfirmationMessageLabel;
 
     @Inject
-    @DataField("confirm-ok-button-text")
-    Span okButton;
+    @DataField("confirm-cancel")
+    Button cancelButton;
+
+    @Inject
+    @DataField("confirm-ok")
+    Button okButton;
 
     @Inject
     @DataField("modal")
     private Modal modal;
 
+    @Inject
+    private TranslationService translationService;
+
     private Command okCommand;
 
+    @PostConstruct
+    public void init() {
+        cancelButton.setText(translationService.getTranslation(Constants.ConfirmPopup_Cancel));
+    }
+
     public void show(String title,
                      String okButtonText,
                      String confirmMessage,
                      final Command okCommand) {
+        show(title,
```

Comment:
Minor thing, could we please use `final` keyword for arguments of both `show` methods. I think it is good to be consistent with style of declaration `@EventHandler` methods in this class.